include/exadg/structure/material/library/st_venant_kirchhoff.h

```diff
@@ -88,6 +96,22 @@ class StVenantKirchhoff : public Material<dim, Number>
   Number
   get_f2_factor() const;
 
+  /*
+   * The second Piola-Kirchhoff stress tensor S is given as S = lambda * I * tr(E) + 2 mu E, with E
+   * being the Green-Lagrange strain tensor and Lamee parameters lambda and mu. This leads to
+   * Sii = f0 * Eii + f1 * Eij, for i, j = 1, ..., dim and i != j, and
```

Comment:
I think here one was to write `sum_{i != j} E_ij`, i.e. the sum is missing, because `S_ii` is not a sum but a single diagonal entry.